### PR TITLE
Server timing headers

### PR DESCRIPTION
**Pull request by Kagaminara**
This MR follows [!3530](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/3530) and [!3537](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/3537) and documents the new ServerTimings service.

- [Migration guide](https://deploy-preview-919--heuristic-almeida-1a1f35.netlify.app/docs/3.x/upgrade/migration-guides/3.5-3.6#server-timings)
- [Services](https://deploy-preview-919--heuristic-almeida-1a1f35.netlify.app/docs/3.x/api-reference/front-commerce-core/services#server-services)
- [Guide](https://deploy-preview-919--heuristic-almeida-1a1f35.netlify.app/docs/3.x/guides/adding-your-own-server-timings)

### CODE REVIEW

File: docs/02-guides/adding-your-own-server-timings.mdx [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+title: Add your own server timings
+description:
+  This guide explains how you can add `Server-Timing` headers in your responses
+  in Front-Commerce.
```
Comment (paul-vd) [inline docs/02-guides/adding-your-own-server-timings.mdx:5]: Not sure if this is accurate? `Server-Timing` headers will always be there?, instead this guide explains how to add new timings in the existing `Server-Timing` header?

File: docs/02-guides/adding-your-own-server-timings.mdx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+---
+
+<SinceVersion tag="3.6" />
+
+The `Server-Timing` header communicates one or more metrics and descriptions for
+a given request-response cycle. In Front-Commerce, this header is handled by the
+`Server-Timing` service.
+
+In this guide, we will show you how to add to use this to surface backend server
+timing metrics in order to diagnose potential performance issues.
```
Comment (paul-vd) [inline docs/02-guides/adding-your-own-server-timings.mdx:19]:
```suggestion
In this guide, we will show you how to use this to surface backend server
timing metrics in order to diagnose potential performance issues.
```

File: docs/02-guides/assets/server-timing-graphql.png
Comment (paul-vd) [inline docs/02-guides/assets/server-timing-graphql.png:1]: Should we not see `acmeLoader.init` in this image?

**Review by paul-vd — APPROVED** [review]
I don't think I need to review again unless you want me too.

File: docs/02-guides/assets/server-timing-graphql.png
Comment (Kagaminara) [reply docs/02-guides/assets/server-timing-graphql.png:1]: Indeed, I uploaded two times the same picture it seems.

**Pull request merged**